src/components/PatronBlock/PatronBlockModalWithOverrideModal.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import OverrideModal from './OverrideModal';
+import PatronBlockModal from './PatronBlockModal';
+
+class PatronBlockModalWithOverrideModal extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      additionalInfo: '',
+      overrideModal: false,
+    };
+  }
+
+  onSetAdditionalInfo = (value = '') => {
+    this.setState({ additionalInfo: value });
+  };
+
+  onOpenOverrideModal = () => {
+    this.setState({ overrideModal: true });
+  };
+
+  onCloseOverrideModal = () => {
+    this.setState({ overrideModal: false });
+  };
+
+  render() {
+    const {
+      additionalInfo,
+      overrideModal,
+    } = this.state;
+    const {
+      patronBlockedModalOpen,
+      viewUserPath,
+      patronBlocks,
+      onClosePatronBlockedModal,
+      onOpenPatronBlockedModal,
+      onRenew,
+    } = this.props;
+
+    return (
+      <>
+        <OverrideModal
+          open={overrideModal}
+          additionalInfo={additionalInfo}
+          onSetAdditionalInfo={this.onSetAdditionalInfo}
+          onClose={() => {
+            this.onCloseOverrideModal();
+            onOpenPatronBlockedModal();
+          }}
+          onSave={async () => {
+            this.onCloseOverrideModal();
+            this.onSetAdditionalInfo();
```

Comment:
@Dmitriy-Litvinenko is there a reason `additionalInfo` is reset to an empty string here?

Reply:
We need clear current message after we use renew  loan(s) by barcode. For case that we don't have possible for renew loan(s) by barcode we need use current message for override renewal by barcode. This was discussed with PO.